packages/Webkul/Admin/src/Resources/assets/js/components/navigation/nav-slide-button.vue

```diff
@@ -31,9 +37,15 @@ export default {
             let self = this;
 
             $('.aside-nav').hide(function () {
-                $('.content-wrapper').css({
-                    marginLeft: 'unset'
-                });
+                if (self.direction === 'rtl') {
+                    $('.content-wrapper').css({
+                        marginRight: 'unset'
+                    });
+                } else {
+                    $('.content-wrapper').css({
+                        marginLeft: 'unset'
+                    });
+                }
```

Comment:
I think instead of adding extra prop, value can be unset from the scss file as well. If you write unset in the rtl file.